flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/utils/SimpleAckingTaskManagerGateway.java

```diff
@@ -48,6 +48,8 @@
 
 	private Optional<Consumer<ExecutionAttemptID>> optSubmitCondition;
 
+	private Optional<Consumer<ExecutionAttemptID>> optCancelCondition;
```

Comment:
This will be always `null` initialized. Isn't that a problem?

Reply:
Jup it is. Fixed it.